### DISCUSSION TITLE
Coding the 'exit' button

### DISCUSSION THREAD

**Discussion by longjulia** [Jo-Lo/Tafe22S203#4] [General]
Hey guys, 

have you coded up the exit button on your applications so it returns to the main menu? i think we should include the click event so we don't have to edit it again after the "big merge" to the development branch. Try the following code block and let me know if it works for you: 

private void exitButton_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
        {
            // return to the universal calculator menu page 
            this.Frame.Navigate(typeof(GIT_Assignment_Main_Menu.MainMenu));
        }

**Comment by Jo-Lo** [comment]
oh right, good idea.

I have updated my exit button code with your suggested code and I had no errors when running the application.
